tests/components/application-connector/README.md

```diff
@@ -2,65 +2,172 @@
+- [Test runners](./test/application-gateway/) with various check for subsets of cases, grouped by Application CRs
+- [Mock application](./tools/external-api-mock-app/) which simulates remote endpoints
+
+Additionally, following resources are created on the cluster:
+- [Service Account](./resources/charts/gateway-test/templates/service-account.yml:2), used by tests to read Application CRs
+
+The tests are executed as a Kubernetes Job on a Kyma cluster, 
+where the tested Application Gateway is installed. 
+The test job and mock application deployment are in the `test` Namespace. 
 
 ![Application Gateway tests architecture](./assets/app-gateway-tests-architecture.svg)
 
-The test cases are defined as services in the Application CR.
+## Building
 
-The whole test setup is deployed into the Kyma cluster with the Helm template executed in the Makefile by the command `test-gateway`.
+To build **and push** Docker images of tests and mock application:
 
-Image versions and the external service name used during testing can be set up in the Helm chart values file `k8s/gateway-test/values.yaml`.
+``` sh
+./scripts/local-build.sh {DOCKER_TAG} {DOCKER_PUSH_REPOSITORY}
+```
+This will build the following images:
+- `{DOCKER_PUSH_REPOSITORY}/gateway-test:{DOCKER_TAG}`
+- `{DOCKER_PUSH_REPOSITORY}/mock-app:{DOCKER_TAG}`
 
-### Local build of test images
+## Running
 
-1. In the main folder build the test images.
-   ```bash
-   ./scripts/local-build.sh {DOCKER_TAG} {DOCKER_PUSH_REPOSITORY}
-   ```
+Tests can be run on any Kyma cluster with Application Gateway.
 
-### Local execution
+### Deploy Kyma cluster locally
 
 1. Provision a local Kubernetes cluster with k3d:
-
-   ```shell
+   ```sh
    kyma provision k3d
    ```
 
-2. Install the minimal set of components required to run Application Gateway for either Kyma OS or SKR:
+1. Install the minimal set of components required to run Application Gateway for either Kyma OS or SKR:
+
+    <div tabs name="Kyma flavor" group="minimal-kyma-installation">
+    <details open>
+    <summary label="OS">
+    Kyma OS
+    </summary>
+
+    ```sh
+    kyma deploy --components-file ./resources/installation-config/mini-kyma-os.yaml
+    ```
+
+    </details>
+    <details>
+    <summary label="SKR">
+    SKR
+    </summary>
+
+    ```bash
+    kyma deploy --components-file ./resources/installation-config/mini-kyma-skr.yaml 
+    ```
+
+    </details>
+    </div>
+
+    >**TIP:** More on Kyma installation can be found in [official Kyma documentation](https://kyma-project.io/docs/kyma/latest/02-get-started/01-quick-install/#install-kyma)
+
+### Run the tests
+
+``` sh
+make test-gateway
+```
+
+By default tests clean up after themselves, removing all created resources
+and `test` Namespace.
+
+> **CAUTION:** This might override or remove existing resources, 
+> if names of already existing resources collide with names used by tests
```

Comment:
https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#tenses

Reply:
> Question: is there a quick and easy way to check what are the names used by the tests?

Not really, at this moment, but those are constant, so we could just list them. Writing a script to list names also shouldn't take more than few minutes, but if we want to fix it, the best idea would be to add parametrized prefix to names

Reply:
I believe it would be helpful for the user to know what these resources are called in order to avoid naming collision, but I also don't think it has to be done on this PR. You might log it as a follow-up, for example. 

As for the tense, touche :) I edited the suggestion.